server/publications/collections/products.js

```diff
@@ -545,6 +545,27 @@ Meteor.publish("Products", function (productScrollLimit = 24, productFilters, so
   const mediaProductIds = productCursor.fetch().map((p) => p._id);
   const mediaCursor = findProductMedia(this, mediaProductIds);
 
+  const handle = productCursor.observe({
```

Comment:
@prinzdezibel Isn't this equivalent to lines `546` and `571`? All the media from the products and their variants should already be published unless I'm missing something here.